internal/cephfs/controllerserver.go

```diff
@@ -145,7 +145,7 @@ func checkContentSource(
 }
 
 // CreateVolume creates a reservation and the volume in backend, if it is not already present.
-// nolint:gocognit,gocyclo,nestif,cyclop // TODO: reduce complexity
+// nolint:gocyclo,cyclop // needs to get fixed late
```

Comment:
please bring back the TODO as we are tracking TODO's in cephcsi https://www.tickgit.com/browse?repo=github.com/ceph/ceph-csi&branch=devel

Reply:
sure.